covid19_sfbayarea/data/sonoma.py

```diff
@@ -0,0 +1,261 @@
+        deaths.append(death_dict)
+
+        # new_recovered = recoveries - cumul_recovered
+        # recovered.append({ 'date': date, 'recovered': new_recovered, 'cumul_recovered': recoveries })
+        #
+        # new_active = active_cases - cumul_active
+        # active.append({ 'date': date, 'active': new_active, 'cumul_active': active_cases })
+
+    return { 'cases': cases, 'deaths': deaths }
+
+def transform_transmission(transmission_tag: element.Tag) -> Dict[str, int]:
+    """
+    Takes in a BeautifulSoup tag for the transmissions table and breaks it into
+    a dictionary of type:
+    {'community': -1, 'from_contact': -1, 'travel': -1, 'unknown': -1}
+    """
+    transmissions = {}
+    rows = get_rows(transmission_tag)
+    # turns the transmission categories on the page into the ones we're using
+    transmission_type_conversion = {'Community': 'community', 'Close Contact': 'from_contact', 'Travel': 'travel', 'Under Investigation': 'unknown'}
+    for row in rows:
+        type, number, *rest = get_cells(row)
+        if type not in transmission_type_conversion:
+            raise FormatError('The transmission type {0} was not found in transmission_type_conversion'.format(type))
+        type = transmission_type_conversion[type]
+        transmissions[type] = parse_int(number)
+    return transmissions
+
+def transform_tests(tests_tag: element.Tag) -> Dict[str, int]:
+    tests = {}
+    rows = get_rows(tests_tag)
+    for row in rows:
+        result, number, *rest = get_cells(row)
+        lower_res = result.lower()
+        tests[lower_res] = parse_int(number)
+    return tests;
+
+def transform_gender(tag: element.Tag) -> Dict[str, int]:
+    """
+    Transform function for the cases by gender table.
+    Takes in a BeautifulSoup tag for a table and returns a dictionary
+    in which the keys are strings and the values integers
+    """
+    categories = {}
+    rows = get_rows(tag)
+    gender_string_conversions = {'Males': 'male', 'Females': 'female'}
+    for row in rows:
+        gender, cases, *rest = get_cells(row)
+        if gender not in gender_string_conversions:
+            raise FormatError('An unrecognized gender has been added to the gender table')
+        categories[gender_string_conversions[gender]] = parse_int(cases)
+    return categories
+
+def transform_age(tag: element.Tag) -> List[Dict[str, Union[str, int]]]:
+    """
+    Transform function for the cases by age group table.
+    Takes in a BeautifulSoup tag for a table and returns a list of
+    dictionaries in which the keys are strings and the values integers
+    """
+    categories: List[Dict[str, Union[str, int]]] = []
+    rows = get_rows(tag)
+    for row in rows:
+        group, cases, *rest = get_cells(row)
+        raw_cases = parse_int(cases)
+
+        element: Dict[str, Union[str, int]] = {'group': group, 'raw_cases': raw_cases}
```

Comment:
This should be `raw_count`:

```suggestion
        element: Dict[str, Union[str, int]] = {'group': group, 'raw_count': raw_cases}
```